install/kubernetes/helm/istio/values-istio-demo.yaml

```diff
@@ -6,18 +6,17 @@ global:
     accessLogFile: "/dev/stdout"
 
   disablePolicyChecks: false
+  
+  outboundTrafficPolicy:
+    mode: ALLOW_ANY  
 
 pilot:
   traceSampling: 100.0
 
 mixer:
-  policy:
-    enabled: true
```

Comment:
this as well? our examples in docs will break

Reply:
This is just a duplicate from the default profile.  So no change in behavior